rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -2598,6 +2598,45 @@ public JobValidationData validate(String sessionId, PathSegment pathSegment, Mul
         }
     }
 
+    @Override
+    public JobValidationData validateFromUrl(String sessionId, String url, Map<String, String> jsonBody,
+            PathSegment pathSegment) throws NotConnectedRestException {
+
+        File tmpWorkflowFile = null;
+        try {
+            Scheduler scheduler = checkAccess(sessionId);
+            SchedulerSpaceInterface space = getSpaceInterface(sessionId);
+            String jobXml = downloadWorkflowContent(sessionId, url);
+            tmpWorkflowFile = File.createTempFile("job", "d");
```

Comment:
it would be best to remove the usage of this temporary file (and also in the original validateFromUrl method).

you can look at the implementation of submitFromUrl